Cargo.lock

```diff
@@ -619,6 +620,23 @@ dependencies = [
 ]
 
 [[package]]
+name = "ethcore-secstore"
+version = "1.0.0"
+dependencies = [
+ "ethcore-devtools 1.6.0",
+ "ethcore-ipc 1.6.0",
+ "ethcore-ipc-codegen 1.6.0",
+ "ethcore-ipc-nano 1.6.0",
+ "ethcore-util 1.6.0",
+ "ethcrypto 0.1.0",
+ "ethkey 0.2.0",
+ "hyper 0.9.14 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
hyper is a bit heavy and slow, have you considered our async version or for instance `tokio-minihttp`?